### PR TITLE
Update 2024-08-07-community-meeting.mdx

### PR DESCRIPTION
**Pull request by CazJT**
Meeting notes added

### CODE REVIEW

File: community/2024-08-07-community-meeting.mdx [hunk 1]
```diff
@@ -17,41 +17,70 @@ title: 'Meeting Agenda - 2024-08-07'
...
-
-### Topic 2
-
-- Note 1
-- Note 2
+### DEMO: `wash` and `wadm` developer experience improvements
+
+- As always, notes can only tell so much. Check out the recording for the full demo.
+- wasmCloud branch - this is public (prs list).
+- We've changed the was we display tables in `wash` - for longer pieces of information.
```
Comment (ericgregory) [inline community/2024-08-07-community-meeting.mdx]:
```suggestion
- We've changed the way we display tables in `wash` - for longer pieces of information.
```

File: community/2024-08-07-community-meeting.mdx [hunk 2]
```diff
@@ -17,41 +17,70 @@ title: 'Meeting Agenda - 2024-08-07'
...
+- We've also updated the app status command in `wadm` which is a really useful development.
+- If we `run-wash` and `list-apps`, we can see a fresh view. Descriptions are nested in the table underneath. Much easier way to capture this detail.
```
Comment (ericgregory) [inline community/2024-08-07-community-meeting.mdx]:
```suggestion
- If we run `wash app list`, we can see a fresh view. Descriptions are nested in the table underneath. Much easier way to capture this detail.
```

File: community/2024-08-07-community-meeting.mdx [hunk 3]
```diff
@@ -17,41 +17,70 @@ title: 'Meeting Agenda - 2024-08-07'
...
+- If we deploy an app, we did not always get the version back from the app. Now we always get the version back, even if we don't specify it.
+- Our new app status command is well worth a look. In `wash` and `wadm` we're now surfacing the top level status info + all the scaler info for each application.
```
Comment (ericgregory) [inline community/2024-08-07-community-meeting.mdx]:
```suggestion
- Our new `wash app status` command is well worth a look. In `wash` and `wadm` we're now surfacing the top level status info + all the scaler info for each application.
```

**Review by brooksmtownsend — COMMENTED** [review]
LGTM! Just waiting on Eric's suggestions I think and good to go

**Review by lachieh — APPROVED** [review]

**Pull request merged**